CONTRIBUTING_NI.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to NI-maintained fork of Grafana
+
+### Branching Policy
+
+Before making any changes, make sure you have read and fully understand the [Branching Policy](https://github.com/ni/grafana/wiki/Branching-policy).
+
+### Changes
```

Comment:
Thanks for mentioning the focus on staying close to the mainline and trying to upstream things first when possible.